core/server/data/schema/schema.js

```diff
@@ -643,5 +643,21 @@ module.exports = {
         entry_id: {type: 'string', maxlength: 24, nullable: true},
         source_url: {type: 'string', maxlength: 2000, nullable: true},
         metadata: {type: 'string', maxlength: 191, nullable: true}
+    },
+    custom_theme_settings: {
+        id: {type: 'string', maxlength: 24, nullable: false, primary: true},
+        theme: {type: 'string', maxlength: 191, nullable: false},
+        key: {type: 'string', maxlength: 50, nullable: false},
```

Comment:
Is 50 chars enough for custom property names? `@custom.50_snake_case_chars_is_a_fairly_long_property_var`

Reply:
Switched to 191